source/agora/node/BlockStorage.d

```diff
@@ -14,7 +14,7 @@
 
 *******************************************************************************/
 
-module agora.common.BlockStorage;
+module agora.node.BlockStorage;
```

Comment:
Could also move it to a `storage` package in the future.

Reply:
Yep, I was considering `agora.node.network` and `agora.node.storage` as well. But later.